### ISSUE TITLE
about dn_labelbook_size

### ISSUE THREAD

**Issue by Innary** [IDEA-Research/DINO#124]
Hello,
When I  read the code I found：
`self.label_enc = nn.Embedding(dn_labelbook_size + 1, hidden_dim)`
In your model， the loss function is sigmoid_focal_loss， and  dn_labelbook_size and num_classes are 91 when using the coco dataset.
So：
1. Why not use 90 but 91 for dn_labelbook_size and num_classes?
2. Why is the first dimension of label_enc 92 instead of 90?

thank you

**Comment by FengLi-ust** [comment]
Hey, the dimension of label_enc is not important, as COCO only has around 80 classes, but we use `dn_labelbook_size + 1` here. You can simply ignore the empty classes that are not used.

**Comment by Innary** [comment]
think you for your answer
